src/components/Modals/RulesModal.vue

```diff
@@ -254,12 +254,14 @@
                     <li>Select a card to put in the "True" or "False" playing field that was chosen at the beginning of the round, or to discard.</li>
                     <li>At the beginning of all of the player's turns, the game will choose either the "True" or "False" playing field as the "Active" part of each program to run.</li>
                     <li>You get bonus points for good programming practice. For more information on this go to the 'Bonus Points' tab.</li>
-                    <li>The game is over if a player reaches the score limit in either paths:</li>
+                    <li>The game ends when a player reaches the score limit in either path:</li>
```

Comment:
We no longer have two paths, so just say "reaches score limit."

Reply:
Does this refer to the "True" or "False" Playing fields? Because it might make sense for me to adjust those lines at the same time. What do you think?

Reply:
Yes, we no longer have a "true" and "false" playing field.

Reply:
I have adjusted the 'Playing the Game' text to remove the true and false fields. I put in a little information about what to expect with a selected card to replace it.

Reply:
I think maybe I should mention that a player can redraw their hand in these rules as well. It should probably be mentioned in what a person can do on their turn.

Reply:
Changes are made and pushed.